corehq/apps/users/templates/users/base_template.html

```diff
@@ -3,5 +3,5 @@
 {% load hq_shared_tags %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static "locations/js/widgets_main.js" %}"></script>
```

Comment:
Looks like a quotation syntax error

Reply:
I don't understand - what's the error? I thought github diff just doesn't know quite how to highlight these files.

Reply:
Oh cool, then you can disregard, I was just trusting github highlighting!